_spinnaker/operator.md

```diff
@@ -113,44 +145,47 @@ spinnaker-operator-7cd659654b-4vktl      2/2           Running      0
 
 # Installing Spinnaker Using Operator
 
-After you install Operator, you can create a `ConfigMap` and `SpinnakerService` to install Spinnaker.
+After you install the CRDs and Operator, you can create a `ConfigMap` and `SpinnakerService` to install Spinnaker.
```

Comment:
This is no longer true. I'd take the paragraph from the PR: "Once you've installed CRDs and Operator, ..."